docs/docs/hpc-on-k8s/hadoop.md

```diff
@@ -0,0 +1,70 @@
+# Hadoop Workload
+Hadoop is an open-source framework that efficiently processes and stores vast amounts of data across distributed computing clusters. It breaks down large datasets into manageable chunks and distributes them across multiple computers, allowing for parallel processing and analysis. This distributed approach makes it possible to handle massive amounts of data that would be impractical to process on a single machine.
+
+
+## Steps to Execute Job
+Before running workload it is required to setup hadoop cluster.
+```bash
+
+helm repo add pfisterer-hadoop https://pfisterer.github.io/apache-hadoop-helm/
+helm install hadoop --set persistence.nameNode.storageClass=longhorn --set persistence.dataNode.storageClass=longhorn --set persistence.dataNode.size=2Gi --set persistence.nameNode.size=2Gi   pfisterer-hadoop/hadoop
+```
+
+Below is the example manifest file for Hadoop Mapreduce workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: hadoop-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: openflighthpc/hadoop:latest
+        command: [ "python3", "/opt/main.py" ]
+        env:
+          - name: MINIO_IP
+            value: <minio ip like "10.151.15.78">
+          - name: MINIO_PORT
+            value: <minio port like "31100">
+          # Minio Access Key   
+          - name: MINIO_AKEY
+            value: <minio-access-key>
+          # Minio Secret Key   
+          - name: MINIO_SKEY
+            value: <minio-secret-key>
+          - name: BUCKET_NAME
+            value: <minio bucket name like "hadoop">
```

Comment:
Some issues experienced IRL with not quoting strings, perhaps putting the entire `<>` section in quotes would make this a little clearer (especially as the access key and secret key don't have any indication of this being needed)

Reply:
updated thanks